CSharp/Samples/QnAMaker/QnABotWithActiveLearning/Dialogs/QnADialogWithActiveLearning.cs

```diff
@@ -4,7 +4,7 @@
 namespace QnABotWithActiveLearning.Dialogs
 {
     [Serializable]
-    [QnAMaker("set yout subscription key here", "set your kbid here", "I don't understand this right now! Try another query!", 0.50, 3)]
+    [QnAMaker("set yout subscription key here", "set your kbid here", "I don't understand this right now! Try another query!", null, 0.50, 3)]
```

Comment:
You should not need to change this--if you moved the new param to the end it would be fine.  Alternatively, name the args you want to set.